### PR TITLE
Implement toggle status feature

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
#16

### CODE REVIEW

**Comment by hothanhtuan1209** [comment]
image

File: myproject/supermarket_manager/models.py [hunk 1]
```diff
@@ -55,10 +55,17 @@ class Account(models.Model):
...
-    GENDER_CHOICES = [("M", "Male"), ("F", "Female"), ("O", "Other")]
+    GENDER_CHOICES = [
+        ("M", "Male"),
+        ("F", "Female"),
+        ("O", "Other")
+    ]
     gender = models.CharField(max_length=1, choices=GENDER_CHOICES)
 
-    STATUS_CHOICES = [("A", "Active"), ("D", "Disable")]
+    STATUS_CHOICES = [
```
Comment (cuongdoan-agilityio) [inline myproject/supermarket_manager/models.py:65]: Please create the constant to re-use

**Comment by cuongdoan-agilityio** [comment]
> image

The image is not correct.

**Comment by cuongdoan-agilityio** [comment]
Please try to combine the features update permission and toggle status

File: myproject/supermarket_manager/models.py (hunk 1 above)
Comment (hothanhtuan1209) [reply myproject/supermarket_manager/models.py:65]: Done

File: myproject/supermarket_manager/views.py [hunk 2]
```diff
@@ -191,14 +192,21 @@ def update_permission(request, permission_id):
...
         try:
             permission = Permission.objects.get(permission_id=permission_id)
             data = json.loads(request.body)
 
+            if "status" in data:
+                if permission.status == "A":
+                    permission.status = "D"
+                else:
+                    permission.status = "A"
+                permission.save()
```
Comment (cuongdoan-agilityio) [inline myproject/supermarket_manager/views.py:208]: Please double check this line

Comment (hothanhtuan1209) [reply myproject/supermarket_manager/views.py]: Done

**Pull request merged**